src/Service/PdfGeneratorService.php

```diff
@@ -0,0 +1,35 @@
+<?php
+
+// src/Service/PdfGeneratorService.php
+
+namespace App\Service;
+
+use Dompdf\Dompdf;
+use Dompdf\Options;
+use Twig\Environment;
+
+class PdfGeneratorService
+{
+    private $twig;
+
+    public function __construct(Environment $twig)
+    {
+        $this->twig = $twig;
+    }
+
+    public function generatePdf(string $template, array $data, string $outputPath): void
+    {
+
+        $html = $this->twig->render($template, $data);
+        $options = new Options();
+        // $options->set('isRemoteEnabled', true);
```

Comment:
remove dead code

Reply:
Sometimes dead code can be helpful for debugging